challenge_one/src/agent.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  ethers,
+} from "forta-agent";
+
+export const NETHERMIND_FORTA_ADDRESS: string =
+  "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8";
+
+export const UPDATE_AGENT_FUNCTION: string =
+  "function updateAgent(uint256 agentId,string metadata,uint256[] chainIds)";
+export const CREATE_AGENT_FUNCTION: string =
+  "function createAgent(uint256 agentId,address owner,string metadata,uint256[] chainIds)";
+
+export const provideHandleTransaction =
+  (): HandleTransaction =>
+  async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
+
+    if (NETHERMIND_FORTA_ADDRESS === txEvent.from) {
```

Comment:
```suggestion
    if (NETHERMIND_FORTA_ADDRESS.toLowerCase() !== txEvent.from) {
```
This should be not-equal. Right now, the test transaction of the README passes because `txEvent.from` is in lower case.